try {
  var 1 = 1;
} catch (IOException ex) {
}